components/private/Shapes/docs/README.md

```diff
@@ -0,0 +1,70 @@
+# Shapes Core
+
+Our Shapes Core library consists of different APIs and classes we use to build shapes for our components. 
+Concretely, to create a shape object, you'll need to create an object that implements the `MDCShapeGenerating` protocol. 
+`MDCShapeGenerating` only has one method, `(CGPath *)pathForSize:(CGSize)size` that returns the shape’s `CGPath` for the expected size.
+
+Our components support shapes by providing a property called `id<MDCShapeGenerating> shapeGenerator` and applies the provided shape to the component.
+
+## Related Documentation
+
+* [Shape Library](../../ShapeLibrary)
+
+<!-- toc -->
+
+- - -
+
+## Overview
+
+Listed below are classes that can be used to create an object that implements the `MDCShapeGenerating` protocol.
```

Comment:
Suggestion: "This library consists of several classes that are often used together to implement Shape behavior on a component."